test/quantization/core/test_quantized_module.py

```diff
@@ -1082,6 +1088,9 @@ def test_linear_api(self, batch_size, in_features, out_features, use_bias, use_d
         """test API functionality for nn.quantized.dynamic.Linear"""
         W = torch.rand(out_features, in_features).float()
         W_scale, W_zp = _calculate_dynamic_qparams(W, torch.qint8)
+        # ONEDNN only supports symmetric quantization of weight
+        if torch.backends.quantized.engine == 'onednn' and W_zp != 0:
```

Comment:
same here

Reply:
Here weight scale and zero point are calculated not selected manually. Do you mean we need a new function to calculate weight scale/zero point for symmetric quantization?

Reply:
Yes I think so, we can add a qscheme argument to https://github.com/pytorch/pytorch/blob/master/torch/testing/_internal/common_quantized.py#L49 to support symmetric quantization and set the qscheme to symmetric quantization when the qengine is mkldnn/onednn